package/python-hvac/python-hvac.hash

```diff
@@ -0,0 +1 @@
+sha256	641ea4ef8aaabf3b7bf6616434b97fa7e03066b2ee8dbb0d97467603a97e300a  hvac-0.10.4.tar.gz
```

Comment:
# Locally computer line above? unless this was not?

And to upstream - you normally need to hash the LICENSE file to whatever.